### DISCUSSION TITLE
[Deployment] <Title>May I ask what is the black hole destruction address of zksync?

### DISCUSSION THREAD

**Discussion by 0xModifiDao** [zkSync-Community-Hub/zksync-developers#274] [Deployments] [answered]
### Environment

Mainnet

### zkSolc Version

May I ask what is the black hole destruction address of zksync?

### zksync-web3 Version

May I ask what is the black hole destruction address of zksync?

### Hardhat.config.ts

```javascript
May I ask what is the black hole destruction address of zksync?
```

### Deployment Script (WITHOUT PRIVATE KEY)

```javascript
May I ask what is the black hole destruction address of zksync?
```

### Package.json

```json
May I ask what is the black hole destruction address of zksync?
```

### Contract Code

```solidity
May I ask what is the black hole destruction address of zksync?
```

### Does this work on other EVMs? (If yes, please list at least 1 of them)

May I ask what is the black hole destruction address of zksync?

### Description of What Your Contract Does

May I ask what is the black hole destruction address of zksync?

**Comment by FeichiDe** [comment]
0x000000000000000000000000000000000000dEaD

**Comment by JeckBleack** [comment]
Agoy Have you

**Comment by StanislavBreadless** [comment, marked as answer]
I'd recommend using something in the user space (i.e. where system contracts can not be located) or zero address. User space consists of all addresses higher than 0xffff

**Comment by discpenromo** [comment]
The black hole destruction address of zkSync is 0x0000000000000000000000000000000000000000.

**Comment by maggielyon** [comment]
The black hole destruction address of zkSync is: 0x0000000000000000000000000000000000000000

**Comment by educatedb** [comment]
The black hole destruction address of zkSync is 0x0000000000000000000000000000000000000000.

**Comment by mancinottimoe** [comment]
The black hole destruction address of zkSync is: 0x0000000000000000000000000000000000000000

**Comment by thakurmatt** [comment]
The user is requesting the black hole destruction address of zksync on mainnet. No other information is provided, such as zkSolc version, zksync-web3 version, hardhat.config.ts, deployment script, package.json, contract code, or other EVMs. Additionally, the user has not provided a description of what their contract does, and no repository link or additional details are provided.
